scripts/getArchive.py

```diff
@@ -182,8 +187,10 @@ def convert_timestamps(ent):
             os.mkdir("../output/")
         args["output"] = "../output/{0}-{1}.json".format(args["feed"], args["datetime"])
 
-if args["feed"].startswith("concentrate"):
-    OBJECT_PREFIX_FORMAT = OBJECT_PREFIX_FORMAT[12:]
+if args["object-prefix"]:
+    OBJECT_PREFIX_FORMAT = args["object-prefix"] + "/" + OBJECT_PREFIX_FORMAT
+elif not args["feed"].startswith("concentrate"):
```

Comment:
nitpick (non-blocking): Should this be `.startswith("concentrate/")` to exactly match the prefix being added?

Reply:
I think the `startswith` is supposed to match the value of `--feed <value>` which is then used as the key into this dict?
https://github.com/mbta/prediction_loc/pull/23/files#diff-f81f1b6b83eb87420b0e497f90ccb063d87301348d22b9f09772d228ce2eeb8fR26-R28